### ISSUE TITLE
Specifying base country

### ISSUE THREAD

**Issue by MaybeThisIsRu** [rwieruch/purchasing-power-parity#8]
Hi, thanks for the lovely API!
I poked around but it seems the data used to calculate the PPP details is not open-source. What would you need from the community to improve on the limitation of USD being the default base? Wondered if I could help.

**Comment by lunchboxav** [comment]
yes, I second this one too. I was looking for how the calculation was done but could not find it. perhaps that way, we can fix #9 too. :)

**Comment by brianmuimi** [comment]
Is this API still under development?
